src/infra/service/graph.js

```diff
@@ -31,16 +34,41 @@ class GraphList {
         }
       });
   }
+  
+  fetchWithToken(url: string, accessToken: string) {
+    return getWithToken(url, accessToken)
+      .then(res => {
+        this.data = [...this.data, ...res.data];
+        if (res.paging) {
+          this.previous = res.paging.previous;
+          this.next = res.paging.next;
+        } else {
+          this.previous = '';
+          this.next = '';
+        }
+      });
+  }
 
-  fetchForward(pages: number = 10, currentPage: number = 1) {
-    return this.fetch(currentPage === 1 ? this.url : this.next)
+  fetchForward(url: string, accessToken: string = '', pages: number = 10, currentPage: number = 1) {
```

Comment:
same as before, don't expose `access token`, let `window.FB` handle it internally.